### PR TITLE
Index page & sorting functions

### PR DESCRIPTION
**Pull request by ValuableRow**
Please review!

### CODE REVIEW

File: app/assets/stylesheets/components/_button.scss [hunk 1]
```diff
@@ -16,12 +16,12 @@
   color: white;
   padding: 8px 24px;
   border-radius: 4px;
-  background: #670BFF;
+  background: #198754;
```
Comment (MNicehouse) [inline app/assets/stylesheets/components/_button.scss]:
```suggestion
  background: #670BFF; 
```

File: app/assets/stylesheets/components/_button.scss [hunk 2]
```diff
@@ -16,12 +16,12 @@
...
   transition: background 0.3s ease;
 }
 
 .btn-flat:hover {
-  background: #4D04C4;
+  background: #198754;
```
Comment (MNicehouse) [inline app/assets/stylesheets/components/_button.scss]:
```suggestion
  background: #4D04C4;
```

**Review by MNicehouse — CHANGES_REQUESTED** [review]
Looks well. Just a few change in app/assets/stylesheets/components/_button.scss

**Comment by ValuableRow** [comment]
@MNicehouse please merge pull request now!!!!

**Pull request merged**